bam_mate.c

```diff
@@ -166,6 +184,18 @@ static void sync_mq(bam1_t* src, bam1_t* dest)
 
         bam_aux_append(dest, "MQ", 'i', sizeof(uint32_t), (uint8_t*)&mq);
     }
+    // Always copy Mate Cigar it can have meaning even in unmapped reads (hard clipping etc)
```

Comment:
I'm not convinced on the usefulness of always adding this.  It may be helpful when one read is mapped and the other is "placed" (ie it is unmapped, but has chr/pos along side the mate), but I don't see this as being useful when both ends are unmapped, with chr \* and pos 0.

Right now it appears that such data gets MC:Z:\* added.  Is that deliberate?